failing to build master

Hello, I am having a problem building ndbench on a machine that has never built it. The gradlew build is throwing errors. Perhaps this is a documentation error or a configuration error on my part.

```
java version "1.8.0_162"
Java(TM) SE Runtime Environment (build 1.8.0_162-b12)
Java HotSpot(TM) 64-Bit Server VM (build 25.162-b12, mixed mode)
```

○ → git clone git@github.com:Netflix/ndbench.git
```Cloning into 'ndbench'...
remote: Counting objects: 2936, done.
remote: Compressing objects: 100% (4/4), done.
remote: Total 2936 (delta 3), reused 1 (delta 1), pack-reused 2931
Receiving objects: 100% (2936/2936), 1.15 MiB | 1.90 MiB/s, done.
Resolving deltas: 100% (809/809), done.
```
○ → cd ndbench

○ →  ls
```CONTRIBUTING.md            build.gradle               gradle                     manifest.yml               ndbench-dynamodb-plugins   ndbench-geode-plugins      ndbench-web
LICENSE                    buildViaTravis.sh          gradlew                    ndbench-api                ndbench-dyno-plugins       ndbench-janusgraph-plugins settings.gradle
OSSMETADATA                codequality                gradlew.bat                ndbench-cass-plugins       ndbench-es-plugins         ndbench-proxy.js
README.md                  dynomite.yml               img                        ndbench-core               ndbench-evcache-plugins    ndbench-sample-plugins
```
**○ →  ./gradlew clean build**
```
> Configure project :
Inferred project: ndbench, version: 0.4.0-SNAPSHOT

Download http://dax-sdk.s3-website-us-west-2.amazonaws.com/java/DaxJavaClient-latest.jar
Publication nebula not found in project :.
Publication named 'nebula' does not exist for project ':' in task ':artifactoryPublish'.

> Task :ndbench-core:compileJava
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.


FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all files for configuration ':ndbench-dynamodb-plugins:compileClasspath'.
> Could not find :DaxJavaClient-latest:.
  Searched in the following locations:
      file:/Users/pivotal/workspace/ndbench/ndbench-dynamodb-plugins/build/DaxJavaClient-latest.jar
  Required by:
      project :ndbench-dynamodb-plugins

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 6s
24 actionable tasks: 23 executed, 1 up-to-date
```

Instead just trying a build....

**○ →  ./gradlew build**
```
> Configure project :
Inferred project: ndbench, version: 0.4.0-SNAPSHOT

Download http://dax-sdk.s3-website-us-west-2.amazonaws.com/java/DaxJavaClient-latest.jar
Publication nebula not found in project :.
Publication named 'nebula' does not exist for project ':' in task ':artifactoryPublish'.

> Task :ndbench-evcache-plugins:compileJava FAILED
/<home>/ndbench/ndbench-evcache-plugins/src/main/java/com/netflix/ndbench/plugin/evcache/EVCacheTestPlugin.java:36: error: EVCacheTestPlugin is not abstract and does not override abstract method writeBulk(List<String>) in NdBenchClient
public class EVCacheTestPlugin implements NdBenchClient {
       ^
1 error


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':ndbench-evcache-plugins:compileJava'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 9s
21 actionable tasks: 10 executed, 11 up-to-date
```

--- a/ndbench-api/src/main/java/com/netflix/ndbench/api/plugin/NdBenchClient.java
+++ b/ndbench-api/src/main/java/com/netflix/ndbench/api/plugin/NdBenchClient.java
@@ -49,15 +49,18 @@ import java.util.List;
      * @return
      * @throws Exception
      */
-    List<String> readBulk(final List<String> keys) throws Exception;
+    default List<String> readBulk(final List<String> keys) throws Exception {
+        throw new UnsupportedOperationException("bulk operation is not supported");
+    }
 
     /**
 	 * Perform a bulk write operation
 	 * @return
 	 * @throws Exception
 	 */
-	List<String> writeBulk(final List<String> keys) throws Exception;
-
+    default List<String> writeBulk(final List<String> keys) throws Exception {
+        throw new UnsupportedOperationException("bulk operation is not supported");
+    }
 
 	/**
      * shutdown the client

--- a/ndbench-evcache-plugins/build.gradle
+++ b/ndbench-evcache-plugins/build.gradle
@@ -9,8 +9,8 @@ repositories {
 }
 
 dependencies {
+        compile project(':ndbench-api')
         compile group:"com.netflix.evcache",          name:"evcache-client",                   version:"4.+"
         compile group:"net.spy",                      name:"spymemcached",                     version:"2.11.4"
-        compile group:"com.netflix.ndbench",          name:"ndbench-api",                      version:"latest.release"
         compile group:"org.slf4j",                    name:"slf4j-api",                        version:"1.7.+"
 }

